pkg/signup/service/signup_service.go

```diff
@@ -171,9 +171,18 @@ func (s *ServiceImpl) GetSignup(userID string) (*signup.Signup, error) {
 		VerificationRequired: userSignup.Spec.VerificationRequired,
 	}
 	if mur.Status.UserAccounts != nil && len(mur.Status.UserAccounts) > 0 {
-		// TODO Set ConsoleURL in UserSignup.Status. For now it's OK to get it from the first embedded UserAccount status from MUR.
-		signupResponse.ConsoleURL = mur.Status.UserAccounts[0].Cluster.ConsoleURL
-		signupResponse.CheDashboardURL = mur.Status.UserAccounts[0].Cluster.CheDashboardURL
+		// Retrieve Console and Che dashboard URLs from the status of the corresponding member cluster
+		status, err := s.CRTClient().V1Alpha1().ToolchainStatuses().Get()
+		if err != nil {
+			return nil, errors2.Wrap(err, fmt.Sprintf("error when retrieving ToolchainStatus to set Che Dashboard for completed UserSignup %s", userSignup.GetName()))
```

Comment:
I believe you could use `errors2.Wrapf` to avoid the use of `fmt.Sprintf`, but it's not critical

Reply:
Yep. Changed it in https://github.com/codeready-toolchain/registration-service/pull/141/commits/7f1d753c9461c654e2fe787a9164e73c45d9225a